lib/loader.js

```diff
@@ -221,7 +221,11 @@ Loader.prototype.prepFixtureData = function(data, Model) {
             if (Model.attributes.hasOwnProperty(key)) {
                 result[key] = val;
             } else {
-                console.warn('attribute "' + key +"' not defined on  model '" + Model.name + "'.");
+                if (self.options.throwErrorOnUndefinedAttribute === true) {
```

Comment:
Suggest: `errorOnUndefinedAttribute`.

Also, `if (self.options.throwErrorOnUndefinedAttribute)` is sufficient... anything truthy should be OK.